tests/core/test_grid.py

```diff
@@ -1093,6 +1093,22 @@ def test_grid_points():
     )
 
 
+def test_imagedata_index_to_physical_matrix():
+    # Create image with arbitrary translation (origin) and rotation (direction)
+    image = pv.ImageData()
+    translation = (1, 2, 3)
+    rotation = pv.Transform().rotate_vector(translation, 30).matrix[:3, :3]
+    image.origin = translation
+    image.direction_matrix = rotation
+
+    expected_transform = pv.Transform(rotation).translate(translation)
```

Comment:
better to use a more generic variable name (it's a bit confusing to have a var named `translate` inside of `rotate` method)
```suggestion
    vector = (1, 2, 3)
    rotation = pv.Transform().rotate_vector(vector, 30).matrix[:3, :3]
    image.origin = translation
    image.direction_matrix = rotation

    expected_transform = pv.Transform(rotation).translate(vector)
```